### ISSUE TITLE
Feature Request: Add support for opening the sidebar in a Firefox container

### ISSUE THREAD

**Issue by n8marti** [matthewruzzi/Google-Chat-Firefox-Sidebar#2]
I would like to use this add-on, but I manage several Google accounts and keep them isolated within their own Firefox containers. Would it be possible to add support for choosing which Firefox container is used when using the sidebar? I'm assuming it currently just opens in no container.

**Comment by pataquets** [comment]
@n8marti Did you figure out any workaround? In case you want to track it, I've filed #3 with a similar problem, which might help you if it's solved.

**Comment by n8marti** [comment]
Not really--I've switched to using the Thunderbird AddOn, but I would rather go back to the Firefox sidebar if this feature gets added
